docs/frontend/architecture/module-structure.md

```diff
@@ -0,0 +1,158 @@
+# Module structure
+
+### Introduction
+
+A single module in its construction corresponds to a typical Nuxt-based project structure.<br>
+By using the [VueMS][vuems] library we can build modules like small projects based on [NuxtJS][nuxt].<br>
+Modules are based on mechanisms [Nuxt Modules][nuxt-modules], but they have no restrictions on the structure. <br>
+The module can have any structure, which means it can be a single functionality or a large and complex business logic.<br>
+We divide modules at our discretion and it is also our decision what structure they will have.<br>
```

Comment:
We divide modules at our discretion  - o co Ci tutaj chodzi? że dzielimy się modułami otwarcie?